assets/css/custom.css

```diff
@@ -0,0 +1,29 @@
+.title-header {padding: .75rem 1.25rem; background-color: #f5f5f5; border-bottom: 1px solid transparent;}
+.title-header h3 {font-size: 0.80rem; margin: 0;}
+.movies {margin-top: 2rem;}
+.img-card {width: 100%; margin-bottom: .40rem;}
+.movies {margin-bottom: .60rem;}
+.series {margin-bottom: .60rem;}
+.footer {padding: 1rem 0; margin-top: 2rem; font-size: 80%; text-align: left;}
+.footer p {margin: 0;}
+.footer-links {padding-left: 0; margin-bottom: 1rem;}
+.footer-links li {display: inline-block;}
+.footer a {font-weight: 500; color: inherit;}
+.footer-links li + li {margin-left: 1rem;}
+.navbar {display: block;}
+body {
+    padding-top: 1.5rem;
+    padding-bottom: 1.5rem;
+}
+.header {
+    padding-bottom: 1rem;
+    border-bottom: .05rem solid #e5e5e5;
+}
+#loadmore {
+    margin-top: 30px;
+    display: none;
+}
+
+.single_card_div {
```

Comment:
Naming consistency is also important. You've used `kebab-case`, `nocase` and `snake_case` on the file.

Reply:
Yes, for sure. Again.. I got this CSS template in some forum and and i didn't improve it.